### PR TITLE
cal3d, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Not tested on x86_64 (yet) ...

File: media-libs/cal3d/cal3d-0.120.recipe [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=12.0.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	cal3d$secondaryArchSuffix = $portVersion
+	cmd:cal3d_converter$secondaryArchSuffix
+	lib:libcal3d$secondaryArchSuffix = libVersionCompat
```
Comment (alaviss) [inline media-libs/cal3d/cal3d-0.120.recipe]: :smile: 

```diff
-	lib:libcal3d$secondaryArchSuffix = libVersionCompat
+	lib:libcal3d$secondaryArchSuffix = $libVersionCompat
```

File: media-libs/cal3d/cal3d-0.120.recipe [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	"
+
+defineDebugInfoPackage cal3d$secondaryArchSuffix \
+	"$binDir"/cal3d_converter \
+	"$libDir"/libcal3d.so.$libVersion
+
+PATCH()
+{
+	cd cal3d
+	sed -i 's/AM_USE_UNITTESTCPP/#\ AM_USE_UNITTESTCPP/' configure.in
```
Comment (alaviss) [inline media-libs/cal3d/cal3d-0.120.recipe]: Is this still necessary with unittest++ updated?

File: media-libs/cal3d/cal3d-0.120.recipe [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+COPYRIGHT="2001-2006 Mekensleep
+	Laurent Desmecht
+	Loic Dachary
+	Bruno Heidelberger"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mp3butcher/Cal3D/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="fce2f57128bbefbcd4cee2d45337165e94614bc839c84e9c22b9bd988f10a440"
+SOURCE_FILENAME="cal3d-$portVersion.tar.gz"
+SOURCE_DIR="Cal3D-$portVersion"
```
Comment (alaviss) [inline media-libs/cal3d/cal3d-0.120.recipe]:
```diff
-SOURCE_DIR="Cal3D-$portVersion"
+SOURCE_DIR="Cal3D-$portVersion/cal3d"
```

We can use this to drop `cd` statements if the build system doesn't use any files outside of the said directory.

File: media-libs/cal3d/cal3d-0.120.recipe (hunk 2 above)
Comment (Begasus) [reply media-libs/cal3d/cal3d-0.120.recipe]: updated the version for unittest++, but cal3d still doesn't pick it up (iirc it checks with pkg-config for unittest++.pc butt unittest++ provides UnitTest++.pc) ...

File: media-libs/cal3d/cal3d-0.120.recipe (hunk 3 above)
Comment (Begasus) [reply media-libs/cal3d/cal3d-0.120.recipe]: Should the patchset be rewritten? haikuporter is smart enough to use the correct path though ...

File: media-libs/cal3d/cal3d-0.120.recipe (hunk 2 above)
Comment (Begasus) [reply media-libs/cal3d/cal3d-0.120.recipe]: build fails when trying to include unittest++ ... got as far as "testResults_" not declared ...

**Pull request merged**